### PR TITLE
In Student.java, the description of courseID is not correct #3183 

### LINKED ISSUE #3183: In Student.java, the description of courseID is not correct
screenshot_8

### PR DESCRIPTION
**Pull request by chanjunweimy**
Fixed #3183 
editted the description of courseID in Student.java

Signed-off-by: Chan Jun Wei <chanjunweimy@gmail.com>

### CODE REVIEW

**Comment by kanghj** [comment]
@chanjunweimy, follow the naming convention and link to the issue from the pull request description. 
https://github.com/TEAMMATES/repo/blob/master/devdocs/process.md

**Comment by chanjunweimy** [comment]
@kanghj  Thank you for your reply and sorry for the inconvenience caused. Is this ok, now? :)

File: src/main/java/teammates/storage/entity/Student.java [hunk 1]
```diff
@@ -40,7 +40,8 @@
     private String email;
 
     /**
-     * The student's Google ID. Used as the foreign key for the Course object.
+     * The student's Course ID. References the primary key of the course.
+     * This shows the course that is currently taking/taken by the student.
```
Comment (kanghj) [inline src/main/java/teammates/storage/entity/Student.java]: Rephrase this line? You can omit "currently".

**Comment by chanjunweimy** [comment]
@kanghj  I have rephrased that line. Does it sound better now?

**Comment by kanghj** [comment]
@chanjunweimy, looks good. :)

**Comment by chanjunweimy** [comment]
@kanghj  Thanks a lot :)

**Pull request merged**